getting-started/Installation/nuget-packages.md

```diff
@@ -11,27 +11,32 @@ position: 6
 
 # NuGet Packages 
 
-Telerik Document Processing provides the __RadPdfProcessing__, __RadSpreadProcessing__, __RadSpreadStreamProcessing__, __RadWordsProcessing__, and __RadZipLibrary__ UI-independent and cross-platform libraries which enable you to process content between different formats and work with archive files. Currently, the libraries are not shipped independently, and they are part of the products that provide UI components (UI for ASP.NET, Kendo UI, UI for WPF, UI for WinForms, UI for Xamarin, UI for Blazor).   
- 
+Telerik provides NuGet packages with the assemblies for all five Document Processing libraries: __RadPdfProcessing__, __RadSpreadProcessing__, __RadSpreadStreamProcessing__, __RadWordsProcessing__, and __RadZipLibrary__. These UI-independent cross-platform libraries enable you to process and convert content in various formats and even work with archive files. We deliver these libraries as a complement to the Telerik UI component suites (UI for ASP.NET, Kendo UI, UI for WPF, UI for WinForms, UI for Xamarin, UI for Blazor), and you don't need to purchase an additional license to use them.
```

Comment:
Let's open the article with the information that we provide NuGet packages for the libraries. The introductory paragraph doesn't make this clear.